.azure-pipelines/ci-build.yml

```diff
@@ -156,21 +162,12 @@ extends:
               SessionTimeout: 20
 
           # arguments are not parsed in DotNetCoreCLI@2 task for `pack` command, that's why we have a custom pack command here
-          - pwsh: dotnet pack $env:BUILD_SOURCESDIRECTORY/src/Microsoft.Kiota.Abstractions.csproj /p:IncludeSymbols=true /p:SymbolPackageFormat=snupkg --no-build --output $env:BUILD_ARTIFACTSTAGINGDIRECTORY --configuration $env:BUILD_CONFIGURATION
+          - pwsh: dotnet pack /p:IncludeSymbols=true /p:SymbolPackageFormat=snupkg --no-build --output $env:BUILD_ARTIFACTSTAGINGDIRECTORY --configuration $env:BUILD_CONFIGURATION
             env:
               BUILD_CONFIGURATION: $(BuildConfiguration)
             displayName: Dotnet pack
 
-          - task: PowerShell@2
-            displayName: 'Validate project version has been incremented'
-            condition: and(contains(variables['build.sourceBranch'], 'refs/heads/main'), succeeded())
-            inputs:
-              targetType: 'filePath'
-              filePath: $(System.DefaultWorkingDirectory)\scripts\ValidateProjectVersionUpdated.ps1
-              arguments: '-projectPath "$(Build.SourcesDirectory)/src/Microsoft.Kiota.Abstractions.csproj" -packageName "Microsoft.Kiota.Abstractions"'
-              pwsh: true
-
-          - task: EsrpCodeSigning@2
+          - task: EsrpCodeSigning@3
```

Comment:
assuming we align version numbers and bump everyone all the time like in the other languages, we could still use abstractions for validation

Reply:
Just to confirm here, the version validation is still present and works as you mentioned(validating the with the Abstractions version). 
I just moved it up in the file so that we don't have the various stages running unnecessarily if this step was going to fail anyway.

Reply:
ah ok, I missed that, thanks!